src/main/java/employee/controller/EnterApplicationController.java

```diff
@@ -0,0 +1,94 @@
+package employee.controller;
```

Comment:
Why is this in the employee package? It would be better in the home package

Reply:
What is the Employee modification for?

Reply:
I added toasts to various Employee actions